cognite/neat/_rules/exporters/_rules2excel.py

```diff
@@ -130,8 +137,96 @@ def export(self, rules: VerifiedRules) -> Workbook:
+        """Adds drop down menus to specific columns for fast and accurate data entry.
+
+        Args:
+            workbook: Workbook representation of the Excel file.
+            no_rows: number of rows to add drop down menus. Defaults to 500.
+        """
+
+        self._make_helper_sheet(workbook)
+
+        dv_views = generate_data_validation("_helper", "A", no_header_rows=0, no_rows=no_rows)
+        dv_containers = generate_data_validation("_helper", "b", no_header_rows=0, no_rows=no_rows)
+        dv_value_types = generate_data_validation("_helper", "C", no_header_rows=0, no_rows=no_rows)
+        dv_immutable = generate_data_validation("_helper", "D", no_header_rows=0, no_rows=3)
+        dv_nullable = generate_data_validation("_helper", "D", no_header_rows=0, no_rows=3)
+        dv_is_list = generate_data_validation("_helper", "D", no_header_rows=0, no_rows=3)
+        dv_in_model = generate_data_validation("_helper", "D", no_header_rows=0, no_rows=3)
+        dv_used_for = generate_data_validation("_helper", "E", no_header_rows=0, no_rows=3)
+
+        workbook["Properties"].add_data_validation(dv_views)
+        workbook["Properties"].add_data_validation(dv_containers)
+        workbook["Properties"].add_data_validation(dv_value_types)
+        workbook["Properties"].add_data_validation(dv_nullable)
+        workbook["Properties"].add_data_validation(dv_is_list)
+        workbook["Properties"].add_data_validation(dv_immutable)
+        workbook["Views"].add_data_validation(dv_in_model)
+        workbook["Containers"].add_data_validation(dv_used_for)
+
+        if column := find_column_with_value(workbook["Properties"], "View"):
+            dv_views.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Properties"], "Container"):
+            dv_containers.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Properties"], "Value Type"):
+            dv_value_types.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Properties"], "Nullable"):
+            dv_nullable.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Properties"], "Is List"):
+            dv_is_list.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Properties"], "Immutable"):
+            dv_immutable.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Views"], "In Model"):
+            dv_in_model.add(f"{column}{3}:{column}{no_rows}")
+
+        if column := find_column_with_value(workbook["Containers"], "Used For"):
+            dv_used_for.add(f"{column}{3}:{column}{no_rows}")
+
+    def _make_helper_sheet(self, workbook: Workbook) -> None:
+        """This helper sheet is used as source of data for drop down menus creation"""
+        workbook.create_sheet(title="_helper")
```

Comment:
Suggest making the helper sheet a private class variable. 
```suggestion
self._helper_sheet
```